crypto/src/merkle/mod.rs

```diff
@@ -0,0 +1,39 @@
+pub mod merkle_path_set;
+pub mod merkle_tree;
+
+use winterfell::crypto::Hasher as HashFn;
+
+pub use winterfell::crypto::hashers::Rp64_256 as Hasher;
+pub use winterfell::math::{
+    fields::{f64::BaseElement as Felt, QuadExtension},
+    ExtensionOf, FieldElement, StarkField,
+};
+
+// TYPE ALIASES
+// ================================================================================================
+
+pub type Word = [Felt; 4];
```

Comment:
`Word` is already defined in `lib.rs`- so, we can probably just import it from there.